### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,4 @@
 language: dart
 dart:
-  - dev
-sudo: false
+  - dev/release/1.25.0-dev.16.0
```
Comment (devoncarew) [inline .travis.yml]: Why a specific release? We'll need to bump this frequently.

Comment (kevmoo) [reply .travis.yml]: Lining up with https://github.com/dart-lang/dart-services/blob/master/dart-sdk.version

Comment (devoncarew) [reply .travis.yml]: makes sense. can you add a comment to this file that when changing one version we should change the other? We'll want the same comment in the other location, but we'll probably need to extend the parser a bit to handle comments. Can you open an issue for that (I'll take a look at it at some point)?

Alternatively we could have a build process to validate the versions are kept in sync.

**Comment by kevmoo** [comment]
PTAL

File: lib/src/sdk_manager.dart [hunk 2]
```diff
@@ -52,11 +52,14 @@ class HostSdk extends Sdk {
...
+  final String _versionFull;
 
-  DownloadingSdk() {
-    _versionFull = new File('dart-sdk.version').readAsStringSync().trim();
-  }
+  DownloadingSdk()
+      : _versionFull = new File('dart-sdk.version')
+            .readAsLinesSync()
+            .map((line) => line.trim())
+            .where((line) => line.isNotEmpty && !line.startsWith('#'))
```
Comment (devoncarew) [inline lib/src/sdk_manager.dart]: 👍 awesome

**Pull request merged**